app/src/main/java/com/github/ybecker/epforuml/AnswerAdapter.kt

```diff
@@ -55,30 +59,79 @@ class AnswerAdapter(private val question: Model.Question, private var anonymouse
         return question.answers.size + HEADER_ITEM_COUNT
     }
 
-
     private fun displayImageFromFirebaseStorage(imageUrl: String, imageView: ImageView) {
         Glide.with(imageView.context)
             .load(imageUrl)
             .transition(DrawableTransitionOptions.withCrossFade())
             .into(imageView)
     }
 
+    private fun displayVideoFromFirebaseStorage(videoUrl: String, videoView: PlayerView) {
+        // create and start a new player from video url
+        val player = videoView.player ?: SimpleExoPlayer.Builder(videoView.context).build()
+        videoView.player = player
+        player.setMediaItem(MediaItem.fromUri(videoUrl))
+        player.prepare()
+        player.play()
+    }
+
     override fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int) {
```

Comment:
It would have been nice to separate a bit more the function logically, for example a new function when it's an image and another one when it's a video